tensorflow/lite/kernels/BUILD

```diff
@@ -1944,7 +1944,7 @@ cc_test(
         "//tensorflow/lite/schema:schema_fbs",
         "@com_google_googletest//:gtest",
         "@flatbuffers",
-    ],
+    ]
```

Comment:
I think this change was unintentional, please remove.

Reply:
ok